### PR TITLE
feat(settings): User settings page implementation

### PR DESCRIPTION
**Pull request by e11sy**
this is the first part of implementation of the `User settings` page 
next time will do list of editor tools in user settings page

image

- user settings page first step of implementation
- style of the Fieldset component updated
- codestyle of the logo usage in header improved

### CODE REVIEW

File: src/presentation/pages/Settings.vue [hunk 1]
```diff
@@ -1,5 +1,23 @@
 <template>
-  <h1>{{ t('userSettings.title') }}</h1>
+  <div :class="$style['page-header']">
+    <h1>{{ t('userSettings.title') }}</h1>
+  </div>
```
Comment (neSpecc) [inline src/presentation/pages/Settings.vue:4]: use Heading component

File: src/presentation/pages/Settings.vue [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+  <Fieldset title="General">
```
Comment (neSpecc) [inline src/presentation/pages/Settings.vue:5]: i18n missed

File: src/presentation/pages/Settings.vue [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+    <div :class="$style['page-header__general-fields']">
+      <Section
+        title="name"
```
Comment (neSpecc) [inline src/presentation/pages/Settings.vue:8]: i18n

File: src/presentation/pages/Settings.vue [hunk 4]
```diff
@@ -1,5 +1,23 @@
...
+        :caption="t('userSettings.nameCaption')"
+      >
+        <Row :title="user?.name!" />
+      </Section>
+      <Section
+        title="email"
```
Comment (neSpecc) [inline src/presentation/pages/Settings.vue:14]: i18n

File: src/presentation/pages/Settings.vue [hunk 5]
```diff
@@ -1,5 +1,23 @@
...
+  <div :class="$style['page-header']">
+    <h1>{{ t('userSettings.title') }}</h1>
+  </div>
+  <Fieldset title="General">
+    <div :class="$style['page-header__general-fields']">
+      <Section
+        title="name"
+        :caption="t('userSettings.nameCaption')"
+      >
+        <Row :title="user?.name!" />
```
Comment (neSpecc) [inline src/presentation/pages/Settings.vue:11]: what will on page refresh?

Comment (e11sy) [reply src/presentation/pages/Settings.vue:11]: all fine

File: src/application/i18n/messages/en.json [hunk 6]
```diff
@@ -16,7 +16,12 @@
     "changeTheme": "Change theme",
     "userEditorTools": "User editor tools",
     "uninstallEditorTool": "Uninstall",
-    "toolUninstallConfirmation": "Do you really want to delete this tool?"
+    "toolUninstallConfirmation": "Do you really want to delete this tool?",
+    "nameCaption": "Will be displayed in collaborators list",
+    "emailCaption": "Used for login",
+    "name": "name",
```
Comment (neSpecc) [inline src/application/i18n/messages/en.json:22]:
```suggestion
    "name": "Name",
```

File: src/application/i18n/messages/en.json [hunk 7]
```diff
@@ -16,7 +16,12 @@
...
+    "email": "email",
```
Comment (neSpecc) [inline src/application/i18n/messages/en.json:23]:
```suggestion
    "email": "Email",
```

**Review by neSpecc — APPROVED** [review]

**Review by slaveeks — APPROVED** [review]

**Pull request merged**